easybuild/easyconfigs/a/astropy/astropy-1.0.6-goolf-1.4.10-Python-2.7.3.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = "PythonPackage"
+
+name = 'astropy'
+version = '1.0.6'
+
+homepage = 'http://www.astropy.org/'
+description = """The Astropy Project is a community effort to develop 
+a single core package for Astronomy in Python and foster interoperability 
+between Python astronomy packages."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.3'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('setuptools', '18.4', versionsuffix),
```

Comment:
@tloizou easyconfig for this is missing, any idea if it's available somewhere else, e.g. a PR that isn't merged yet?